block/submit.go

```diff
@@ -297,3 +321,31 @@ func (m *Manager) UpdateLastSubmittedHeight(event pubsub.Message) {
 		}
 	}
 }
+
+// SetLastSettlementBlockTime saves the last block on SL timestamp
+func (m *Manager) SetLastSettlementBlockTime(time time.Time) error {
+	_, err := m.Store.SaveLastSettlementBlockTime(uint64(time.UTC().UnixNano()), nil)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// GetLastSettlementBlockTime returns the last block on  SL timestamp
+func (m *Manager) GetLastSettlementBlockTime() (time.Time, error) {
```

Comment:
why is this function necessary? cant we just call the load block time directly?

Reply:
remved